cmd/argot/main.go

```diff
@@ -22,7 +22,7 @@ func main() {
 	kubeconfig := flag.String("kubeconfig", "", "Path to a kubeconfig file")
 	printVersion := flag.Bool("version", false, "prints application version")
 	namespace := flag.String("namespace", controller.SecretNamespaceDefault, "Namespace to run in")
-	ingressClass := flag.String("ingressClass", controller.SecretNameDefault, "Name of ingress class, used in ingress annotation")
+	ingressClass := flag.String("ingressClass", controller.IngressClassDefault, "Name of ingress class, used in ingress annotation")
```

Comment:
Wow I'm a jerk, lets split this off from the main change set to focus the PR on error handling.
 - https://github.com/cloudflare/cloudflare-ingress-controller/pull/71

Reply:
i'm the only one that should apologize :)